crontab

```diff
@@ -1,3 +1,5 @@
-# sample scan of quarantine directory and subdirectories
-# the scan will run at 00:00 on the first of every month
-0 0 1 * * /usr/local/bin/uvscan --summary --atime-preserve --analyze -r --secure quarantine/data/
+# scan uploads directory on the first day of every month at midnight
+0 0 1 * * /usr/local/bin/uvscan --summary --atime-preserve --analyze -r --secure uploads/
```

Comment:
I think we need to update this to have the full path to the uploads folder. That way we can make sure it will actually find it.